cmd/lakectl/cmd/log.go

```diff
@@ -66,6 +66,19 @@ func (d *dotWriter) Write(commits []apigen.Commit) {
 	}
 }
 
+// filter merge commits, used for --no-merges flag
+func filterMergeCommits(commits []apigen.Commit) []apigen.Commit {
+	filteredCommits := make([]apigen.Commit, 0, len(commits))
```

Comment:
I don't think you can really rely on the length of commits, all you know is that `0<= len(filteredCommits)<=len(commits)`. 
It's really an extremely minor performance issue, but I rather not allocate more than I need
```suggestion
	var filteredCommits []apigen.Commit
```

Reply:
That was my initial proposal, @arielshaqed suggested that